opg-wordpress-1604/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM registry.service.opg.digital/opg-php-fpm-1604
+
+ARG PHP_PACKAGES="php-mysql php-curl libpng12-dev libjpeg-dev"
+
+RUN apt update && \
+    apt install -y $PHP_PACKAGES && \
+    apt clean && apt -y autoremove && \
+    rm -rf /var/lib/cache/* /var/lib/log/* /tmp/* /var/tmp/*
+
+RUN chown app:app /data
+
+USER app
+
+# Install wordpress
+RUN cd /tmp && \
+    wget --no-verbose https://en-gb.wordpress.org/latest-en_GB.tar.gz && \
+    tar -xf latest-en_GB.tar.gz && \
+    rm -rf wordpress/wp-content/themes/* && \
+    rm -rf wordpress/wp-content/plugins/hello.php wordpress/wp-content/plugins/akismet && \
+    mv wordpress/* /data/. && \
+    rm -rf wordpress latest-en_GB.tar.gz
+
+# Add wordpress configuration templates
+ADD docker/confd /etc/confd
```

Comment:
It's recommended using COPY over ADD. 
https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile